Assets/Editor/BuildTiltBrush.cs

```diff
@@ -59,9 +59,10 @@ static class BuildTiltBrush
     // OSX Executable
     public const string kGuiBuildOsxExecutableName = kGuiBuildExecutableName + ".app";
     // Android Application Identifier
-    public static string GuiBuildAndroidApplicationIdentifier => $"foundation.{kVendorName}.{kGuiBuildExecutableName}".ToLower();
+    public static string GuiBuildAndroidApplicationIdentifier => $"{kVendorReverseDNS}.{kGuiBuildExecutableName}".ToLower();
     // Android Executable
     public static string GuiBuildAndroidExecutableName => GuiBuildAndroidApplicationIdentifier + ".apk";
+    public static string GuiBuildiOSApplicationIdentifier => $"{kVendorReverseDNS}.{kGuiBuildExecutableName}".ToLower();
```

Comment:
Do we need two variables if they both (iOS and Android) get the same value?

Reply:
I felt this was more appropriate in case of any forks which may have different naming schemes, for example iOS bundles can only have hyphens while android can only have underscores

Reply:
Ok, as long as it's on purpose!